source/Calamari.Tests/Calamari.Tests.csproj

```diff
@@ -10,6 +10,7 @@
     <ApplicationIcon />
     <StartupObject />
     <RuntimeIdentifiers>win-x64;linux-x64;osx-x64;linux-arm;linux-arm64</RuntimeIdentifiers>
+    <LangVersion>7.3</LangVersion>
```

Comment:
Why do we need this @YihaoOct ?

Reply:
Good catch, I added this to make Rider pick up some incompatible syntax in higher versions of C# that will not build against .net 4 such as file-scoped namespaces and so on. But it turned out there are still some syntax that wasn't captured even with this. This means 7.3 is not the correct version to use to guarantee successful compilation for older .net targets.

I will try to find a better version to set to facilitate local development